arm-web/2016-03-01/ResourceProvider.json

```diff
@@ -207,76 +271,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/publishingCredentials": {
```

Comment:
Intentional. This API is not per ARM spec and ARM does not let it through. So its not breaking anyone.